whatToDad_site/whatToDad_app/templates/forum.html

```diff
@@ -1,30 +1,71 @@
 {% extends 'base.html' %} {% block content %}
 
+
+<style>
+
+.rating {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.rating>input {
+  display: none;
+}
+
+.rating>label {
+  position: relative;
+  width: 1em;
+  font-size: 3vw;
+  color: #FFD600;
+  cursor: pointer;
+}
+
+.rating>label::before {
+  content: "\2605";
+  font-size: 3vw;
+  position: absolute;
+  opacity: 0;
+}
+
+.rating>label:hover:before,
+.rating>label:hover~label:before {
+  opacity: 1 !important;
```

Comment:
Using `!important` is usually a sign that you made your CSS overly specific in the wrong place. Is there a way to accomplish this change using specificity without `!important`